### PR TITLE
Announce the competition venue to volunteers

### PR DESCRIPTION
**Pull request by PeterJCLaw**
Also advertise the upcoming tech days.

### CODE REVIEW

**Review by RealOrangeOne — APPROVED** [review]

File: SR2023/2023-01-18-competition-venue.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+Please note that we are not able to reimburse travel or accommodation at the
+competition, though we hope to provide ways for volunteers who are interested to
+group together to save costs.
+
+    [Volunteer at the Competition](https://forms.gle/HHy2f5naCvP46bZ59)
+
+We're also looking for people to help organise the competition. Even if you only
+have a small amount of time available, every contribution helps make the event
+better for the competitors. If you'd like to help, please come along to a
+competition meeting or contact <competiton-team@studentrobotics.org>.
```
Comment (prophile) [inline SR2023/2023-01-18-competition-venue.md:28]: Blocker: needs to be `competition-team@` (this is a typo)

File: SR2023/2023-01-18-competition-venue.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+to: All Volunteers
+subject: Volunteer at the SR2023 Competition
+---
+
+Hello,
+
+# Competition Event
+
+We're thrilled to announce the venue for Student Robotics 2023 -- it will be
+held in person at Southampton University on the weekend 1st April.
```
Comment (prophile) [inline SR2023/2023-01-18-competition-venue.md:11]: the University of Southampton

**Review by prophile — CHANGES_REQUESTED** [review]

File: SR2023/2023-01-18-competition-venue.md (hunk 2 above)
Comment (PeterJCLaw) [reply SR2023/2023-01-18-competition-venue.md:11]: 405923465f4adaa3e54ae703a75531931aa7093f

File: SR2023/2023-01-18-competition-venue.md (hunk 1 above)
Comment (prophile) [reply SR2023/2023-01-18-competition-venue.md:28]: Doesn't fix the issue 🙃

Comment (PeterJCLaw) [reply SR2023/2023-01-18-competition-venue.md]: b3cc3e7bad789b3e3fb730d798a64f840dec8a99

File: SR2023/2023-01-18-competition-venue.md (hunk 2 above)
Comment (PeterJCLaw) [reply SR2023/2023-01-18-competition-venue.md]: 7f302c195b4f1a99bcac239f7af3ea99940cbd82

**Review by prophile — APPROVED** [review]

**Pull request merged**